src/utils/gateway.ts

```diff
@@ -28,6 +28,6 @@ export const getExplorerLink = (
 }
 
 export const isWalletConnectSafeApp = (url: string): boolean => {
-  const WALLET_CONNECT = /wallet-connect/
-  return WALLET_CONNECT.test(url)
+  const WC_APP_URL = 'https://apps-portal.safe.global/wallet-connect'
```

Comment:
Do we have to account for trailing slashes here?

Reply:
No, that's the precise URL of this safe app.